hack/debug.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# We want to supress the hostkey check
+SSH_OPTS='-o StrictHostKeyChecking=no'
```

Comment:
might as well put the `-o PasswordAuthentication=no` in here too. And move it to run_self_via_ssh(); nothing else uses it